### ISSUE TITLE
Question about padding9 of padding_collate function

### ISSUE THREAD

**Issue by ChillTerry** [haleqiu/AirIMU#14]
Hi, 
   really nice work! When reviewing the code, I am confused about the functionality of padding for the acc and gyro at initial state.  Could you clarify how this padding contributes to the functionality or data processing pipeline?
```python
    pad_acc = init_state['rot'].Inv() * iden_acc_vector
    pad_gyro = torch.zeros(B, pad_len, 3, dtype=input_data['dt'].dtype)

    input_data["acc"] = torch.cat([pad_acc, input_data['acc']], dim =1)
    input_data["gyro"] = torch.cat([pad_gyro, input_data['gyro']], dim =1)
```

**Comment by haleqiu** [comment]
The initial hidden state of the LSTM is all zero. We found that the initial estimation of the network may have some "initial bias". So we add this padding to see if this issue can be addressed. There are lots of different arguments of how to use the LSTM. 

We didn't get a chance to investigate how to remove this effect and whether this effect is problematic. Let me know if you find anything.
